The appclient.bat script needs more changes as per Windows failure:
{quote}
java.lang.AssertionError: Expected an exit value 0f 0 got 1
Command Executed:
C:\BuildAgent\work\e34a6f994de9f7c6\testsuite\scripts\target\wildfly\bin\appclient.bat "-v"
Environment:
Output:
Calling "C:\BuildAgent\work\e34a6f994de9f7c6\testsuite\scripts\target\wildfly\bin\appclient.conf.bat"
'"!DIRNAME!common.bat"' is not recognized as an internal or external command,
operable program or batch file.
Error: Could not find or load main class !JAVA_OPTS!
Caused by: java.lang.ClassNotFoundException: !JAVA_OPTS!
java.lang.AssertionError:
Expected an exit value 0f 0 got 1
Command Executed:
C:\BuildAgent\work\e34a6f994de9f7c6\testsuite\scripts\target\wildfly\bin\appclient.bat "-v"
Environment:
Output:
Calling "C:\BuildAgent\work\e34a6f994de9f7c6\testsuite\scripts\target\wildfly\bin\appclient.conf.bat"
'"!DIRNAME!common.bat"' is not recognized as an internal or external command,
operable program or batch file.
Error: Could not find or load main class !JAVA_OPTS!
Caused by: java.lang.ClassNotFoundException: !JAVA_OPTS!
{quote}

--- a/ee-feature-pack/galleon-shared/src/main/resources/content/bin/appclient.bat
+++ b/ee-feature-pack/galleon-shared/src/main/resources/content/bin/appclient.bat
@@ -116,8 +116,10 @@ if "x%JBOSS_MODULEPATH%" == "x" (
   set  "JBOSS_MODULEPATH=%JBOSS_HOME%\modules"
 )
 
+setlocal EnableDelayedExpansion
 call "!DIRNAME!common.bat" :setSecurityManagerDefault
 set "JAVA_OPTS=!JAVA_OPTS! !SECURITY_MANAGER_CONFIG_OPTION!"
+setlocal DisableDelayedExpansion
 
 rem Set the module options
 set "MODULE_OPTS="

